lib/curl_req.ex

```diff
@@ -79,30 +85,97 @@ defmodule CurlReq do
         body -> ["-d", body]
       end
 
+    redirect =
+      case req.options do
+        %{redirect: true} -> ["-L"]
```

Comment:
This one we _can_ do, because it is not processed in a step, but in the Req core.

Reply:
You sure? Isn't it a response step with `&Req.Steps.redirect/1`?

Reply:
Ah, I see. I think we are ok adding this here though. It doesn't change the actual request that is being done, just what happens to it afterwards.